lib_xud/src/core/included/XUD_Token_Out_DI.S

```diff
@@ -7,50 +7,39 @@
 .align FUNCTION_ALIGNMENT
 .skip 0
 Pid_Out:
+    #define  XTA_CRCADDRCHECK       xta_ep_Pid_Out_header
     #include "XUD_CrcAddrCheck.S"
     ldw        r3, r5[r10]                      // Load relevant EP pointer
     bf         r3, XUD_TokenOut_BufferFull
     ldw        r1, r3[3]                        // Load buffer from EP structure
 
-CheckEpTypeOut:
-    ldw        r11, r3[5]                       // Load EP type
-    BRFT_ru6   r11, DoOutNonIso                 // ISO endpoint
-
 OutReady:
     BLRF_u10    doRXData                        // Leaves r1: 0
-    {clre;
-    ldw         r11, r3[1]}                     // Load EP chanend
-
-InformEP_Iso:                                   // Iso EP - no handshake
-    out        res[r11], r4;                    // Output datalength (words)
-    stw         r1, r5[r10]                     // Clear ready (r1: 0)
-    {outt       res[r11], r8;   ldw    r6, sp[STACK_RXCRC_INIT]} // CRC16 init (out) - Needs reseting after an out & Send tail length
-#if defined(__XS2A__)
-    ldw         r1, sp[STACK_VTOK_PORT]
-#endif
-    #include "XUD_TokenJmp.S"
-
-.align FUNCTION_ALIGNMENT
-.skip 0
+RxRet2:
```

Comment:
Not sure why this extra label is required, but maybe I missed it's usage

Reply:
only for xta purposes. rxloop can return to both setup (RxRet1) and out (RxRet2)